sdk/dotnet/Compute/InstanceTemplate.cs

```diff
@@ -253,8 +253,8 @@ public partial class InstanceTemplate : Pulumi.CustomResource
         public Output<ImmutableArray<Outputs.InstanceTemplateDisk>> Disks { get; private set; } = null!;
 
         /// <summary>
-        /// Enable [Virtual Displays](https://cloud.google.com/compute/docs/instances/enable-instance-virtual-display#verify_display_driver) on this instance.
-        /// **Note**: `allow_stopping_for_update` must be set to true in order to update this field.
+        /// ) Enable [Virtual Displays](https://cloud.google.com/compute/docs/instances/enable-instance-virtual-display#verify_display_driver) on this instance.
+        /// **Note**: `allow_stopping_for_update` must be set to true in order to update this field. Note: This attribute is currently in beta and will be removed from the google provider. Please use the google-beta provider to continue using this attribute.
```

Comment:
This comment would seem to be erroneous since I believe we're in the Google Beta provider, but I am assuming this is an upstream thing.